dds_web/api/project.py

```diff
@@ -545,12 +545,12 @@ def delete(self):
         return {"removed": True}
 
     @staticmethod
-    def delete_project_contents(project):
+    def delete_project_contents(project, delete_bucket=False):
         """Remove project contents"""
         # Delete from cloud
         with ApiS3Connector(project=project) as s3conn:
             try:
-                s3conn.remove_bucket()
+                bucket = s3conn.remove_bucket_contents(delete_bucket=delete_bucket)
```

Comment:
not sure I understand this: delete_project_contents does not seem to return anything and "bucket" is not used anywhere later on.

Reply:
Ah, this was something I was trying out and then forgot to remove it.

Reply:
```suggestion
                s3conn.remove_bucket_contents(delete_bucket=delete_bucket)
```